api/src/utils/get-permissions.ts

```diff
@@ -21,7 +22,13 @@ export async function getPermissions(accountability: Accountability, schema: Sch
 	const cacheKey = `permissions-${hash({ user, role, app, admin, share_scope })}`;
 
 	if (env.CACHE_PERMISSIONS !== false) {
-		const cachedPermissions = await getSystemCache(cacheKey);
+		let cachedPermissions = null;
```

Comment:
```suggestion
		let cachedPermissions;
```

Thoughts on _not_ setting it as null, similar to the code you have mentioned in your PR? 

https://github.com/directus/directus/blob/0122c4da4b2cdabd0fed830f27fbec0caa24071d/api/src/middleware/cache.ts#L23-L31

Of course I don't think there's any issue with `= null` 👍 just slightly gravitating towards not setting null personally, so wondering what are the general consensus around this 😄

Reply:
Good catch :) Lets keep it consistent!